ggml-cuda.cu

```diff
@@ -2841,8 +2841,21 @@ GGML_CALL static bool ggml_backend_cuda_supports_op(ggml_backend_t backend, cons
         case GGML_OP_ARANGE:
         case GGML_OP_TIMESTEP_EMBEDDING:
         case GGML_OP_LEAKY_RELU:
+            return true;
         case GGML_OP_FLASH_ATTN_EXT:
+#if defined(GGML_USE_HIPBLAS) && defined(__HIP_PLATFORM_AMD__)
+            return op->src[0]->ne[0] == 64 || op->src[0]->ne[0] == 128;
+#else
+            if (op->src[0]->ne[0] == 64 || op->src[0]->ne[0] == 128) {
+                return true;
+            }
+            for (int id = 0; id < ggml_backend_cuda_get_device_count(); ++id) {
+                if (ggml_cuda_info().devices[id].cc < CC_VOLTA) {
+                    return false;
+                }
+            }
```

Comment:
I don't think it is necessary to check every device here, instead get the context and check only the device for this context. Something like this:
```C++
ggml_backend_cuda_context * cuda_ctx = (ggml_backend_cuda_context *)backend->context;
if (ggml_cuda_info().devices[ctx->device].cc < CC_VOLTA) {
    return false;
}
```